soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -4745,9 +4737,90 @@ void DrawRandoEditor(bool& open) {
     ImGui::End();
     }*/
 
+typedef struct {
+    GetItemID giid;
+    ItemID iid;
+    std::string english;
+    std::string german;
+    std::string french;
+} GetItemMessage;
```

Comment:
is there a reason this is defined here or would it make more sense in `CustomMessageTypes.h`? (honest question not sure which makes more sense in this case)

Reply:
It might make more sense in `CustomMessageTypes.h`. That file was created after the first batch of reviews from you, due to type redefinition errors when `message_data_static.h` was included in certain files. I pulled the bits from that file I was actually using into `CustomMessageTypes.h`. That particular struct you called out is also likely going to go away if we add support for a syntax for item icons, along with `CreateGetItemMessage` as a whole.

Reply:
coolcool, even if we plan on not using it in the future i think it'd be nice to have it in CustomMessageTypes so it doesn't get lost when we refactor away from needing it (`randomizer.cpp` is really big)